docs/components/Sliders.md

```diff
@@ -74,61 +58,46 @@ You can add icons to the continuous slider like this:
 OdsSlider(
```

Comment:
This example use the old API.

Same comment for `OdsSliderLockups` below and for discrete sliders (4 examples).